Our.Umbraco.TagHelpers/ImgTagHelper.cs

```diff
@@ -314,12 +314,63 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             // The configuration allows us to define whether images are configured "mobile first". This simply alternates between min-width & max-width media queries.
             var imageSizes = GetImageSizes(MediaItem != null);
 
+            // Avoid rendering a WebP alternative if the image is a GIF
+            var imageFormat = MediaItem != null ? MediaItem.Url().Split('.').Last() : FileSource?.Split('.').Last(); 
+            var renderWebP = imageFormat != "gif";
```

Comment:
This should probably be case insensitive, use `imageFormat.Equals("gif", StringComparison.OrdinalIgnoreCase)`.